.dockerignore

```diff
@@ -0,0 +1,13 @@
+# hidden files and directories
+.*
+.*/
+
+# files
+tezos-reward.service_template
+service_add.py
+*.csv
```

Comment:
We can also ignore all *.md files like the readme.md.

Reply:
OK. Fixed

Reply:
Fixed too